frontend/src/routes/index.svelte

```diff
@@ -220,8 +220,31 @@
                     {/if}
                 </div>
             {:else}
-                <div class="flex justify-around">
-                    <p>No results for: {input}</p>
+                <div class="flex flex-col">
+                    <div class="m-auto text-center max-w-xl">
+                        <p>No results for:</p>
+                        <p class="pb-1"><b><i>{input}</i></b></p>
+                        {#if $currentGenres.length > 0 && $currentProviders.length > 0}
+                            <p>With genres:</p>
+                            {#each $currentGenres as genre}
+                                <div class="badge mx-1">{genre}</div>
+                            {/each}
+                            <p class="pt-1">And providers:</p>
+                            {#each $currentProviders as provider}
+                                <div class="badge mx-1">{provider}</div>
+                            {/each}
+                        {:else if $currentGenres.length > 0}
+                            <p>With genres:</p>
+                            {#each $currentGenres as genre}
+                                <div class="badge mx-1">{genre}</div>
+                            {/each}
+                        {:else if $currentProviders.length > 0}
+                            <p>With providers:</p>
+                            {#each $currentProviders as provider}
+                                <div class="badge mx-1">{provider}</div>
+                            {/each}
+                        {/if}
+                    </div>
```

Comment:
Instead of having an `if` and two `else if`, could we make it into two `if`s. Maybe even doing it with input
Pseudocode:
```svelte
<h4>General helper message<h4/>
{#if input}
<p>With input</p>
{/if}
{#if genres}
<p>With genres</p>
{/if}
{#if providers}
<p>With providers</p>
{/if}
```
We will lose the 'And', but I think it's much more readable

Reply:
True i have edited it